connectors/src/connectors/slack/index.ts

```diff
@@ -702,6 +746,123 @@ export class SlackConnectorManager extends BaseConnectorManager<SlackConfigurati
   }
 }
 
+/**
+ * Retrieves Slack client credentials from either customer-specific credentials or environment variables.
+ */
+async function getSlackClientCredentials(
+  configuration: SlackConfigurationResource,
+  connectorId: number
```

Comment:
Doesn't SlackConfigurationResource have the connector id? Why pass it separately?